### PR TITLE
removing Unite 2015 announcement

### PR DESCRIPTION
**Pull request by ryanschmidt**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
       <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
   </head>
-  <body class="announcement-active">
```
Comment (cvan) [inline public/index.html]: don't you want to remove the `.announcement-active` styles too?

Comment (ryanschmidt) [inline public/index.html]: No. They will be adding announcements in the future. I didn't want the markup hanging out on the page but figured the styles were good to stay.

File: public/assets/js/global/main.js [hunk 2]
```diff
@@ -20,9 +20,12 @@
   ga('create', 'UA-49796218-16', 'auto');
   ga('send', 'pageview');
 
-  $('#announcement a').addEventListener('click', function () {
-    ga('send', 'event', 'click.header', 'action-link');
-  });
+  var actionLink = document.getElementById('action-link');
+  if (typeof(actionLink) != 'undefined' && actionLink != null) {
+    $('#action-link').addEventListener('click', function () {
```
Comment (cvan) [inline public/assets/js/global/main.js]: you could just use `actionLink.addEventListener(…)`

File: public/assets/js/global/main.js [hunk 3]
```diff
@@ -20,9 +20,12 @@
   ga('create', 'UA-49796218-16', 'auto');
   ga('send', 'pageview');
 
-  $('#announcement a').addEventListener('click', function () {
-    ga('send', 'event', 'click.header', 'action-link');
-  });
+  var actionLink = document.getElementById('action-link');
+  if (typeof(actionLink) != 'undefined' && actionLink != null) {
```
Comment (cvan) [inline public/assets/js/global/main.js]: could check just `if (actionLink) {`

**Pull request merged**